english/00_8_Agent/The_name_is_Bond_Milton_Bond.md

```diff
@@ -1,6 +1,7 @@
 You have entered into the world of a secret agent.
 If you used to be a Drill Master, you should aspire to become 
-a [White Tiger](https://play.google.com/store/apps/details?id=com.linecorp.LGRGS) instead.
+a [White Tiger](https://play.google.com/store/apps/details?id=com.linecorp.LGRGS) instead. If White Tiger  
+is too dull, you may choose a [Tiger Bear](http://i0.wp.com/listverse.com/wp-content/uploads/2011/05/liger.jpg?resize=548%2C426)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.